tests/python/relay/test_pass_fuse_ops.py

```diff
@@ -287,6 +286,81 @@ def expected(dshape, dtype):
     assert relay.ir_pass.alpha_equal(f, after)
 
 
+def test_gru():
```

Comment:
Note this isn't an actual GRU, it's just some made-up thing, so I'd rename it to test_dense_split_elementwise or something to avoid confusion.

Reply:
Yeah, I was confused, since Googling told me this is not quite the definition of GRU :) Fixed.